packages/common/test/sites/domains/add-domain.test.ts

```diff
@@ -67,6 +67,67 @@ describe("addDomain", function () {
     expect(res.success).toBeTruthy("json parsed and response returned");
   });
 
+  it("converts title to string without client key", async function () {
```

Comment:
For these tests, I'd expect to see passing in the clientKey, and verify that it does not make it into the body of the request (or is present as `""` )